ci-operator/config/openshift/openshift-azure/master.json

```diff
@@ -5,27 +5,33 @@
     "name": "plugin-base"
   },
   "base_images": {
-    "base": {
+    "plugin-base": {
       "cluster": "https://api.ci.openshift.org",
       "namespace": "azure",
       "name": "plugin-base",
       "tag": "latest"
+    },
+    "test-base": {
```

Comment:
You probably don't need this. Revert if so.

Reply:
Isn't enough that you updated `test_base_image`?

Reply:
ok, now I got what you had in mind.